docs/getting-started/rancher.md

```diff
@@ -4,25 +4,34 @@ sidebar_position: 2
 
 # Rancher Setup
 
-## Installing Rancher
+### Installing Rancher
```

Comment:
Q: why is this needed (jumping from h1 to h3)?

Reply:
This was an error. Thanks for noticing! 🙂

Reply:
That’s why a visual inspection on the PR is needed. https://github.com/rancher-sandbox/rancher-turtles-docs/pull/32 Minor change of hn may have a very profound effect on docs visual style, difficult to pickup on the diff alone.